orchestrator/db/models.py

```diff
@@ -85,8 +85,8 @@ def process_result_value(self, value: Optional[datetime], dialect: Dialect) -> O
 class ProcessTable(BaseModel):
     __tablename__ = "processes"
 
-    pid = Column(UUIDType, server_default=text("uuid_generate_v4()"), primary_key=True, index=True)
-    workflow = Column(String(255), nullable=False)
+    process_id = Column("pid", UUIDType, server_default=text("uuid_generate_v4()"), primary_key=True, index=True)
+    workflow_name = Column("workflow", String(255), nullable=False)
```

Comment:
it does not need a migration since it still linked to `"pid"` and `"workflow"`.

Reply:
Yeah, what I meant to suggest was to have a migration to actually rename the columns in the database. But as Maurits suggested when we discussed this, it will break queries that other orchestrator users do on their databases directly. Which is a good point, so let's just keep the DB as-is